pdns/syncres.cc

```diff
@@ -1513,12 +1513,21 @@ vState SyncRes::getDSRecords(const DNSName& zone, dsmap_t& ds, bool taOnly, unsi
 
   if (rcode == RCode::NoError || (rcode == RCode::NXDomain && !bogusOnNXD)) {
 
+    uint8_t bestDigestType = 0;
```

Comment:
It might be cleaner to use a `DNSSECKeeper::dsdigestalgorithm_t` instead.

Reply:
that does mean also adding `#include "dnsseckeeper.hh"` in the syncres

Reply:
`DSRecordContent.d_digesttype` is a `uint8_t`, so this means that we'll need to convert between `uint8_t` and `DNSSECKeeper::dsdigestalgorithm_t` every time. I've kept this a `uint8_t `but will use the enum for comparison.